### PR TITLE
Update some inserts/schemas, SPC version bump (w. fix)

### PR DESCRIPTION
**Pull request by alexkb0009**

### CODE REVIEW

**Comment by j1z0** [comment]
Codacy Here is an overview of what got changed by this pull request:


```diff

Issues
======
+ Solved 8
- Added 12
           
```


See the complete overview on [Codacy](https://app.codacy.com/manual/4dn/fourfront/pullRequest?prid=5753973&bid=18835912)

File: src/encoded/tests/data/inserts/quality_metric_rnaseq_madqc.json [hunk 1]
```diff
@@ -2,68 +2,88 @@
     "uuid": "d0be03da-3700-4148-a225-3852422c121e",
     "award": "1U01CA200059-01",
     "lab": "4dn-dcic-lab",
-    "MAD QC": {
-        "one and two": {
+    "MAD QC" : [
```
Comment (j1z0) [inline src/encoded/tests/data/inserts/quality_metric_rnaseq_madqc.json:5]: Codacy found an issue: [Format Error: unexpected " "](https://app.codacy.com/manual/4dn/fourfront/pullRequest?prid=5753973)

File: src/encoded/tests/data/inserts/quality_metric_rnaseq_madqc.json [hunk 2]
```diff
@@ -2,68 +2,88 @@
...
+            "File 2" : "two",
             "MAD of log ratios": 0.398,
             "Pearson correlation": 0.949,
             "SD of log ratios": 0.573,
             "Spearman correlation": 0.929
         },
-        "one and three": {
+        {
+            "File 1" : "one",
+            "File 2" : "three",
```
Comment (j1z0) [inline src/encoded/tests/data/inserts/quality_metric_rnaseq_madqc.json:16]: Codacy found an issue: [Format Error: unexpected "    "](https://app.codacy.com/manual/4dn/fourfront/pullRequest?prid=5753973)

File: src/encoded/schemas/quality_metric_workflowrun.json [hunk 3]
```diff
@@ -32,6 +32,10 @@
         },
         "url": {
             "qc_order": 2
+        },
+        "Max CPU utilization" : {
+            "type": "number",
+            "qc_order": 3
```
Comment (j1z0) [inline src/encoded/schemas/quality_metric_workflowrun.json:38]: Codacy found an issue: [Format Error: unexpected "     "](https://app.codacy.com/manual/4dn/fourfront/pullRequest?prid=5753973)

File: src/encoded/schemas/quality_metric_workflowrun.json [hunk 4]
```diff
@@ -32,6 +32,10 @@
         },
         "url": {
             "qc_order": 2
+        },
+        "Max CPU utilization" : {
```
Comment (j1z0) [inline src/encoded/schemas/quality_metric_workflowrun.json:36]: Codacy found an issue: [Format Error: unexpected "    "](https://app.codacy.com/manual/4dn/fourfront/pullRequest?prid=5753973)

File: src/encoded/tests/data/inserts/quality_metric_bamqc.json [hunk 5]
```diff
@@ -3,11 +3,12 @@
...
-        "Total Mapped sequences": 38000000,
-        "Total Uniquely Mapped Sequences": 30000000,
-        "%Mapped": 95,
-        "%Uniquely Mapped": 75,
+        "Total Reads": 40000000,
+        "MR": 38000000,
+        "NM" : 200000,
+        "MU": 30000000,
+        "% of reads mapped": 95,
+        "% of reads filtered": 75,
```
Comment (j1z0) [inline src/encoded/tests/data/inserts/quality_metric_bamqc.json:11]: Codacy found an issue: [Format Error: unexpected "    "](https://app.codacy.com/manual/4dn/fourfront/pullRequest?prid=5753973)

File: src/encoded/tests/data/inserts/quality_metric_rnaseq_madqc.json [hunk 6]
```diff
@@ -2,68 +2,88 @@
     "uuid": "d0be03da-3700-4148-a225-3852422c121e",
     "award": "1U01CA200059-01",
     "lab": "4dn-dcic-lab",
-    "MAD QC": {
-        "one and two": {
+    "MAD QC" : [
+        {
+            "File 1" : "one",
```
Comment (j1z0) [inline src/encoded/tests/data/inserts/quality_metric_rnaseq_madqc.json:7]: Codacy found an issue: [Format Error: unexpected "  "](https://app.codacy.com/manual/4dn/fourfront/pullRequest?prid=5753973)

File: src/encoded/tests/data/inserts/quality_metric_rnaseq_madqc.json [hunk 7]
```diff
@@ -2,68 +2,88 @@
...
+            "File 1" : "two",
+            "File 2" : "five",
             "MAD of log ratios": 2.001,
             "Pearson correlation": 0.417,
             "SD of log ratios": 2.356,
             "Spearman correlation": 0.386
         },
-        "three and four": {
+        {
+            "File 1" : "three",
```
Comment (j1z0) [inline src/encoded/tests/data/inserts/quality_metric_rnaseq_madqc.json:63]: Codacy found an issue: [Format Error: unexpected "    "](https://app.codacy.com/manual/4dn/fourfront/pullRequest?prid=5753973)

File: src/encoded/tests/data/inserts/quality_metric_rnaseq_madqc.json [hunk 8]
```diff
@@ -2,68 +2,88 @@
     "uuid": "d0be03da-3700-4148-a225-3852422c121e",
     "award": "1U01CA200059-01",
     "lab": "4dn-dcic-lab",
-    "MAD QC": {
-        "one and two": {
+    "MAD QC" : [
+        {
+            "File 1" : "one",
+            "File 2" : "two",
```
Comment (j1z0) [inline src/encoded/tests/data/inserts/quality_metric_rnaseq_madqc.json:8]: Codacy found an issue: [Format Error: unexpected "  "](https://app.codacy.com/manual/4dn/fourfront/pullRequest?prid=5753973)

File: src/encoded/tests/data/inserts/quality_metric_rnaseq_madqc.json [hunk 9]
```diff
@@ -2,68 +2,88 @@
...
+            "File 2" : "four",
             "MAD of log ratios": 2.013,
             "Pearson correlation": 0.427,
             "SD of log ratios": 2.358,
             "Spearman correlation": 0.401
         },
-        "one and five": {
+        {
+            "File 1" : "one",
+            "File 2" : "five",
```
Comment (j1z0) [inline src/encoded/tests/data/inserts/quality_metric_rnaseq_madqc.json:32]: Codacy found an issue: [Format Error: unexpected "  "](https://app.codacy.com/manual/4dn/fourfront/pullRequest?prid=5753973)

File: src/encoded/schemas/quality_metric_workflowrun.json [hunk 10]
```diff
@@ -32,6 +32,10 @@
...
+            "type": "number",
```
Comment (j1z0) [inline src/encoded/schemas/quality_metric_workflowrun.json:37]: Codacy found an issue: [Format Error: unexpected "   "](https://app.codacy.com/manual/4dn/fourfront/pullRequest?prid=5753973)

File: src/encoded/tests/data/inserts/quality_metric_rnaseq_madqc.json [hunk 11]
```diff
@@ -2,68 +2,88 @@
...
             "MAD of log ratios": 1.973,
             "Pearson correlation": 0.421,
             "SD of log ratios": 2.369,
             "Spearman correlation": 0.391
         },
-        "two and three": {
+        {
+            "File 1" : "two",
```
Comment (j1z0) [inline src/encoded/tests/data/inserts/quality_metric_rnaseq_madqc.json:39]: Codacy found an issue: [Format Error: unexpected " "](https://app.codacy.com/manual/4dn/fourfront/pullRequest?prid=5753973)

File: src/encoded/tests/data/inserts/quality_metric_rnaseq_madqc.json [hunk 12]
```diff
@@ -2,68 +2,88 @@
...
+            "File 1" : "three",
+            "File 2" : "five",
             "MAD of log ratios": 2.006,
             "Pearson correlation": 0.413,
             "SD of log ratios": 2.371,
             "Spearman correlation": 0.384
         },
-        "four and five": {
+        {
+            "File 1" : "four",
```
Comment (j1z0) [inline src/encoded/tests/data/inserts/quality_metric_rnaseq_madqc.json:79]: Codacy found an issue: [Format Error: unexpected " "](https://app.codacy.com/manual/4dn/fourfront/pullRequest?prid=5753973)

File: src/encoded/tests/data/inserts/quality_metric_rnaseq_madqc.json (hunk 2 above)
Comment (alexkb0009) [reply src/encoded/tests/data/inserts/quality_metric_rnaseq_madqc.json:16]: SPACES BEFORE COLONS LOOK PRETTY!!!
Fine.

**Comment by j1z0** [comment]
Codacy Here is an overview of what got changed by this pull request:


```diff

Issues
======
+ Solved 9
- Added 11
           
```


See the complete overview on [Codacy](https://app.codacy.com/manual/4dn/fourfront/pullRequest?prid=5753973&bid=18835912)

File: src/encoded/tests/data/inserts/quality_metric_rnaseq_madqc.json [hunk 13]
```diff
@@ -2,68 +2,88 @@
...
+            "File 2": "four",
             "MAD of log ratios": 2.013,
             "Pearson correlation": 0.427,
             "SD of log ratios": 2.358,
             "Spearman correlation": 0.401
         },
-        "one and five": {
+        {
+            "File 1": "one",
+            "File 2": "five",
```
Comment (j1z0) [inline src/encoded/tests/data/inserts/quality_metric_rnaseq_madqc.json:32]: Codacy found an issue: [Format Error: unexpected "  "](https://app.codacy.com/manual/4dn/fourfront/pullRequest?prid=5753973)

File: src/encoded/schemas/quality_metric_workflowrun.json (hunk 10 above)
Comment (j1z0) [inline src/encoded/schemas/quality_metric_workflowrun.json:37]: Codacy found an issue: [Format Error: unexpected "   "](https://app.codacy.com/manual/4dn/fourfront/pullRequest?prid=5753973)

File: src/encoded/tests/data/inserts/quality_metric_rnaseq_madqc.json [hunk 14]
```diff
@@ -2,68 +2,88 @@
...
+            "File 1": "three",
+            "File 2": "five",
             "MAD of log ratios": 2.006,
             "Pearson correlation": 0.413,
             "SD of log ratios": 2.371,
             "Spearman correlation": 0.384
         },
-        "four and five": {
+        {
+            "File 1": "four",
```
Comment (j1z0) [inline src/encoded/tests/data/inserts/quality_metric_rnaseq_madqc.json:79]: Codacy found an issue: [Format Error: unexpected " "](https://app.codacy.com/manual/4dn/fourfront/pullRequest?prid=5753973)

File: src/encoded/schemas/quality_metric_workflowrun.json (hunk 4 above)
Comment (j1z0) [inline src/encoded/schemas/quality_metric_workflowrun.json:36]: Codacy found an issue: [Format Error: unexpected "    "](https://app.codacy.com/manual/4dn/fourfront/pullRequest?prid=5753973)

File: src/encoded/tests/data/inserts/quality_metric_rnaseq_madqc.json [hunk 15]
```diff
@@ -2,68 +2,88 @@
...
+            "File 1": "two",
+            "File 2": "five",
             "MAD of log ratios": 2.001,
             "Pearson correlation": 0.417,
             "SD of log ratios": 2.356,
             "Spearman correlation": 0.386
         },
-        "three and four": {
+        {
+            "File 1": "three",
```
Comment (j1z0) [inline src/encoded/tests/data/inserts/quality_metric_rnaseq_madqc.json:63]: Codacy found an issue: [Format Error: unexpected "    "](https://app.codacy.com/manual/4dn/fourfront/pullRequest?prid=5753973)

File: src/encoded/tests/data/inserts/quality_metric_rnaseq_madqc.json [hunk 16]
```diff
@@ -2,68 +2,88 @@
...
+            "File 2": "two",
             "MAD of log ratios": 0.398,
             "Pearson correlation": 0.949,
             "SD of log ratios": 0.573,
             "Spearman correlation": 0.929
         },
-        "one and three": {
+        {
+            "File 1": "one",
+            "File 2": "three",
```
Comment (j1z0) [inline src/encoded/tests/data/inserts/quality_metric_rnaseq_madqc.json:16]: Codacy found an issue: [Format Error: unexpected "    "](https://app.codacy.com/manual/4dn/fourfront/pullRequest?prid=5753973)

File: src/encoded/tests/data/inserts/quality_metric_rnaseq_madqc.json [hunk 17]
```diff
@@ -2,68 +2,88 @@
     "uuid": "d0be03da-3700-4148-a225-3852422c121e",
     "award": "1U01CA200059-01",
     "lab": "4dn-dcic-lab",
-    "MAD QC": {
-        "one and two": {
+    "MAD QC": [
+        {
+            "File 1": "one",
+            "File 2": "two",
```
Comment (j1z0) [inline src/encoded/tests/data/inserts/quality_metric_rnaseq_madqc.json:8]: Codacy found an issue: [Format Error: unexpected "  "](https://app.codacy.com/manual/4dn/fourfront/pullRequest?prid=5753973)

File: src/encoded/tests/data/inserts/quality_metric_bamqc.json [hunk 18]
```diff
@@ -3,11 +3,12 @@
...
-        "Total Mapped sequences": 38000000,
-        "Total Uniquely Mapped Sequences": 30000000,
-        "%Mapped": 95,
-        "%Uniquely Mapped": 75,
+        "Total Reads": 40000000,
+        "MR": 38000000,
+        "NM": 200000,
+        "MU": 30000000,
+        "% of reads mapped": 95,
+        "% of reads filtered": 75,
```
Comment (j1z0) [inline src/encoded/tests/data/inserts/quality_metric_bamqc.json:11]: Codacy found an issue: [Format Error: unexpected "    "](https://app.codacy.com/manual/4dn/fourfront/pullRequest?prid=5753973)

File: src/encoded/tests/data/inserts/quality_metric_rnaseq_madqc.json [hunk 19]
```diff
@@ -2,68 +2,88 @@
...
+            "File 1": "one",
+            "File 2": "five",
             "MAD of log ratios": 1.973,
             "Pearson correlation": 0.421,
             "SD of log ratios": 2.369,
             "Spearman correlation": 0.391
         },
-        "two and three": {
+        {
+            "File 1": "two",
```
Comment (j1z0) [inline src/encoded/tests/data/inserts/quality_metric_rnaseq_madqc.json:39]: Codacy found an issue: [Format Error: unexpected " "](https://app.codacy.com/manual/4dn/fourfront/pullRequest?prid=5753973)

File: src/encoded/tests/data/inserts/quality_metric_rnaseq_madqc.json [hunk 20]
```diff
@@ -2,68 +2,88 @@
     "uuid": "d0be03da-3700-4148-a225-3852422c121e",
     "award": "1U01CA200059-01",
     "lab": "4dn-dcic-lab",
-    "MAD QC": {
-        "one and two": {
+    "MAD QC": [
+        {
+            "File 1": "one",
```
Comment (j1z0) [inline src/encoded/tests/data/inserts/quality_metric_rnaseq_madqc.json:7]: Codacy found an issue: [Format Error: unexpected "  "](https://app.codacy.com/manual/4dn/fourfront/pullRequest?prid=5753973)

File: src/encoded/schemas/quality_metric_workflowrun.json (hunk 3 above)
Comment (j1z0) [inline src/encoded/schemas/quality_metric_workflowrun.json:38]: Codacy found an issue: [Format Error: unexpected "     "](https://app.codacy.com/manual/4dn/fourfront/pullRequest?prid=5753973)

**Comment by j1z0** [comment]
Codacy Here is an overview of what got changed by this pull request:


```diff

Issues
======
+ Solved 9
- Added 11
           
```


See the complete overview on [Codacy](https://app.codacy.com/manual/4dn/fourfront/pullRequest?prid=5753973&bid=18835912)

File: src/encoded/tests/data/inserts/quality_metric_rnaseq_madqc.json (hunk 19 above)
Comment (j1z0) [inline src/encoded/tests/data/inserts/quality_metric_rnaseq_madqc.json:39]: Codacy found an issue: [Format Error: unexpected " "](https://app.codacy.com/manual/4dn/fourfront/pullRequest?prid=5753973)

File: src/encoded/tests/data/inserts/quality_metric_rnaseq_madqc.json (hunk 14 above)
Comment (j1z0) [inline src/encoded/tests/data/inserts/quality_metric_rnaseq_madqc.json:79]: Codacy found an issue: [Format Error: unexpected " "](https://app.codacy.com/manual/4dn/fourfront/pullRequest?prid=5753973)

File: src/encoded/tests/data/inserts/quality_metric_rnaseq_madqc.json (hunk 17 above)
Comment (j1z0) [inline src/encoded/tests/data/inserts/quality_metric_rnaseq_madqc.json:8]: Codacy found an issue: [Format Error: unexpected "  "](https://app.codacy.com/manual/4dn/fourfront/pullRequest?prid=5753973)

File: src/encoded/schemas/quality_metric_workflowrun.json (hunk 10 above)
Comment (j1z0) [inline src/encoded/schemas/quality_metric_workflowrun.json:37]: Codacy found an issue: [Format Error: unexpected "   "](https://app.codacy.com/manual/4dn/fourfront/pullRequest?prid=5753973)

File: src/encoded/tests/data/inserts/quality_metric_rnaseq_madqc.json (hunk 20 above)
Comment (j1z0) [inline src/encoded/tests/data/inserts/quality_metric_rnaseq_madqc.json:7]: Codacy found an issue: [Format Error: unexpected "  "](https://app.codacy.com/manual/4dn/fourfront/pullRequest?prid=5753973)

File: src/encoded/schemas/quality_metric_workflowrun.json (hunk 4 above)
Comment (j1z0) [inline src/encoded/schemas/quality_metric_workflowrun.json:36]: Codacy found an issue: [Format Error: unexpected "    "](https://app.codacy.com/manual/4dn/fourfront/pullRequest?prid=5753973)

File: src/encoded/tests/data/inserts/quality_metric_bamqc.json (hunk 18 above)
Comment (j1z0) [inline src/encoded/tests/data/inserts/quality_metric_bamqc.json:11]: Codacy found an issue: [Format Error: unexpected "    "](https://app.codacy.com/manual/4dn/fourfront/pullRequest?prid=5753973)

File: src/encoded/schemas/quality_metric_workflowrun.json (hunk 3 above)
Comment (j1z0) [inline src/encoded/schemas/quality_metric_workflowrun.json:38]: Codacy found an issue: [Format Error: unexpected "     "](https://app.codacy.com/manual/4dn/fourfront/pullRequest?prid=5753973)

File: src/encoded/tests/data/inserts/quality_metric_rnaseq_madqc.json (hunk 13 above)
Comment (j1z0) [inline src/encoded/tests/data/inserts/quality_metric_rnaseq_madqc.json:32]: Codacy found an issue: [Format Error: unexpected "  "](https://app.codacy.com/manual/4dn/fourfront/pullRequest?prid=5753973)

File: src/encoded/tests/data/inserts/quality_metric_rnaseq_madqc.json (hunk 16 above)
Comment (j1z0) [inline src/encoded/tests/data/inserts/quality_metric_rnaseq_madqc.json:16]: Codacy found an issue: [Format Error: unexpected "    "](https://app.codacy.com/manual/4dn/fourfront/pullRequest?prid=5753973)

File: src/encoded/tests/data/inserts/quality_metric_rnaseq_madqc.json (hunk 15 above)
Comment (j1z0) [inline src/encoded/tests/data/inserts/quality_metric_rnaseq_madqc.json:63]: Codacy found an issue: [Format Error: unexpected "    "](https://app.codacy.com/manual/4dn/fourfront/pullRequest?prid=5753973)

**Comment by j1z0** [comment]
Codacy Here is an overview of what got changed by this pull request:


```diff

Issues
======
+ Solved 1
- Added 4
           
```


See the complete overview on [Codacy](https://app.codacy.com/manual/4dn/fourfront/pullRequest?prid=5753973&bid=18835912)

File: src/encoded/schemas/quality_metric_workflowrun.json (hunk 3 above)
Comment (j1z0) [inline src/encoded/schemas/quality_metric_workflowrun.json:38]: Codacy found an issue: [Format Error: unexpected "     "](https://app.codacy.com/manual/4dn/fourfront/pullRequest?prid=5753973)

File: src/encoded/tests/data/inserts/quality_metric_workflowrun.json [hunk 21]
```diff
@@ -4,6 +4,7 @@
         "award": "1U01CA200059-01",
         "lab": "4dn-dcic-lab",
         "url": "someurl",
+        "Max CPU utilization": 99.8,
```
Comment (j1z0) [inline src/encoded/tests/data/inserts/quality_metric_workflowrun.json:7]: Codacy found an issue: [Format Error: unexpected "   "](https://app.codacy.com/manual/4dn/fourfront/pullRequest?prid=5753973)

File: src/encoded/schemas/quality_metric_workflowrun.json (hunk 10 above)
Comment (j1z0) [inline src/encoded/schemas/quality_metric_workflowrun.json:37]: Codacy found an issue: [Format Error: unexpected "   "](https://app.codacy.com/manual/4dn/fourfront/pullRequest?prid=5753973)

File: src/encoded/schemas/quality_metric_workflowrun.json (hunk 4 above)
Comment (j1z0) [inline src/encoded/schemas/quality_metric_workflowrun.json:36]: Codacy found an issue: [Format Error: unexpected "    "](https://app.codacy.com/manual/4dn/fourfront/pullRequest?prid=5753973)

**Pull request merged**